--- a/configure	2025-11-04 20:34:38.000000000 +0000
+++ b/configure	2025-11-04 20:09:35.000000000 +0000
@@ -158,11 +158,13 @@
 
 if [ -n "$NETCDF" ] ; then
   echo "Will use NETCDF in dir: $NETCDF"
+  echo "Will use NETCDFF in dir: $NETCDFF"
   # for 3.6.2 and greater there might be a second library, libnetcdff.a .  Check for this and use
   # if available
-  NETCDFF=" "
-  if [ -f "$NETCDF/lib/libnetcdff.a" ] ; then
-    NETCDFF="-lnetcdff"
+  NETCDFF="$NETCDFF"
+  export NETCDFF
+  if [ -f "$NETCDFF/lib/libnetcdff.a" ] ; then
+    NETCDFF_LIB="-lnetcdff"
   fi
 else
   echo "Will configure for use without NetCDF"
@@ -261,7 +263,7 @@
 fi
 
 # Found perl, so proceed with configuration
-perl arch/Config.pl -perl=$PERL -netcdf=$NETCDF -netcdff=$NETCDFF -os=$os -mach=$mach -wrfdir=$wrf_dir -grib2dir=$grib2dir
+perl arch/Config.pl -perl=$PERL -netcdf=$NETCDF -netcdff=$NETCDFF -netcdfflib=$NETCDFF_LIB -os=$os -mach=$mach -wrfdir=$wrf_dir
 
 
 #Checking cross-compiling capability for some particular environment 
@@ -429,9 +431,9 @@
     FFLAGS=`grep ^FFLAGS configure.wps | cut -d"=" -f2-`
     FORMAT_FREE=`grep ^FORMAT_FREE configure.wps | cut -d"=" -f2-`
     FFLAGS=`printf "%s" "${FFLAGS}" | sed -e "s/\\$(FORMAT_FREE)/${FORMAT_FREE}/g"`
-    cp $NETCDF/include/netcdf.inc .
+    cp $NETCDFF/include/netcdf.inc .
     FC=`grep ^SFC configure.wps | cut -d"=" -f2-`
-    $FC ${FFLAGS} fort_netcdf.f -o fort_netcdf -L${NETCDF}/lib $NETCDFF -lnetcdf > /dev/null 2>&1 
+    $FC ${FFLAGS} fort_netcdf.f -o fort_netcdf -L${NETCDF}/lib -L${NETCDFF}/lib ${NETCDFF_LIB} -lnetcdf > /dev/null 2>&1 
     if [ -f "fort_netcdf" ] ; then
       ./fort_netcdf > /dev/null 2>&1
       if [ $? = 0 ]; then
